### ISSUE TITLE
Server will need BigQuery credentials, which will need to be whitelisted by M-Lab

### ISSUE THREAD

**Issue by JohnTigue** [codeforseattle/seanetmap#27]
https://docs.google.com/document/d/1Y97QwbSIHGN_Q6BaWD5z-Zh_I0uKe95S_9jkDh3fWYY/edit

>Researchers wishing to query the M-Lab dataset should contact support@measurementlab.net to request whitelisting of the Google account(s) used to query with the SDK. Query access is not available until your account is whitelisted.

**Comment by JohnTigue** [comment]
Via email it does seem that the City's instance of SEANetMap will need to have credentials to be accessing BigQuery.

**Comment by brett-miller** [comment]
This is a moot point AFAIK.  Any white listed user should be able to create a service account from the Google API Explorer.

**Comment by JohnTigue** [comment]
Yup. This is a placeholder for the City to decide if they want to use someone's private credentials or, as @critzo was saying during the last meeting, if a service account is to be set up with M-Lab. The details of what that means are unknown to me.

**Comment by critzo** [comment]
@JohnTigue @brett-miller You're both right. This is a google account that is whitelisted by M-Lab. The city can use that account directly against Big Query, or create a service account within Google's developer console to use with the API directly.
